sdks/java/core/src/main/java/org/apache/beam/sdk/testing/RunnableOnService.java

```diff
@@ -17,9 +17,12 @@
  */
 package org.apache.beam.sdk.testing;
 
+import org.apache.beam.sdk.runners.PipelineRunner;
+
 /**
- * Category tag for validation tests which utilize {@link TestPipeline} for execution and
- * {@link PAssert} for validation. Example usage:
+ * Category tag for tests which utilize {@link TestPipeline} for execution and
```

Comment:
This will now be `ValidatesRunner` and I think it should say something about the fact that "this is a test of the runner itself, not (necessarily) the transforms being executed by the runner". Exact wording TBD, and there's the outstanding question of what that means for IO ITs.